docs/source/_templates/layout.html

```diff
@@ -30,8 +30,8 @@
   <meta property="og:title" content="{{ title|striptags|e }}{{ titlesuffix }}">
   <meta name="description" content="{{ theme_variables.og['description'] }}">
   <meta property="og:description" content="{{ theme_variables.og['description'] }}">
-  <meta property="og:image" content="https://mmf.sh/img/logo.png">
-  <meta property="twitter:image" content="https://mmf.sh/img/logo.png">
+  <!--<meta property="og:image" content="https://mmf.sh/img/logo.png">-->
```

Comment:
just curious, why not delete it?

Reply:
I guess I left it as a placeholder so that I wouldn't forget how to implement this feature once we had a logo. The file currently very messy. When I clean it up, I will remove.